zqd/ingest/pcap.go

```diff
@@ -116,7 +140,7 @@ outer:
 			break outer
 		case t := <-ticker.C:
 			if t.After(start.Add(next)) {
-				if err := p.createSnapshot(ctx); err != nil {
+				if err := p.createSnapshot(ctx, p.zeekFiles()); err != nil {
```

Comment:
Would you add a comment stating why the suricata files aren't included in the periodic snapshots?

Reply:
Great question 🙂. This separation is an artefact of how the PR evolved, and as it currently stands there's no reason not to include them. So I've changed this to do that.